From a20ede0649df041ec277f18455a4115470ad0111 Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Thu, 7 Apr 2016 16:37:20 +0800
Subject: [PATCH 1014/5242] MLK-12681 ARM: dts: imx6ull: enable fec1 port in
 ddr3 ARM2 board

commit  bf8e41d706580f98dda9e7243fb6cf1e3cadf93e from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable fec1 port in ddr3 ARM2 board.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index 508d66d..1c69d8b 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -150,7 +150,7 @@
 	pinctrl-0 = <&pinctrl_enet1>;
 	phy-mode = "rmii";
 	phy-handle = <&ethphy0>;
-	status = "disabled";
+	status = "okay";
 };
 
 &fec2 {
-- 
1.7.9.5

